tests/instruments/test_all_instruments.py

```diff
@@ -160,6 +161,10 @@ def find_devices_in_module(module):
     "Keithley2000",
     "Keithley2260B",
     "Keithley2306",
+    "Keithley2306Channel",
+    "BatteryChannel",
+    "Step",
+    "Relay",
```

Comment:
I like to add the corresponding instrument (and file name) to channels, which are otherwise unrecognizable.

```suggestion
    "BatteryChannel",  # Keithley2306
    "Step",  # Keithley2306
    "Relay",  # Keithley2306
```